### PR TITLE
Sync: Add more sync status info to the debugger.

### PR DESCRIPTION
**Pull request by enejb**
Add sync status to the debugger. This info is hopefully useful to the happiness engineers. 

#### Changes proposed in this Pull Request:
Adds data regaridng the sync status to the debug info that users find on the 
http://example.com/wp-admin/admin.php?page=jetpack-debugger

#### Testing instructions:
- Go to the jetpack debugger expand the debug info.

cc: @jeherve 
screen shot 2016-07-27 at 20 54 31

### CODE REVIEW

File: class.jetpack-debugger.php [hunk 1]
```diff
@@ -60,6 +81,26 @@ public static function jetpack_debug_display_handler() {
...
 
+		$debug_info .= "\r\n";
+		require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-modules.php';
+		$sync_module = Jetpack_Sync_Modules::get_module( 'full-sync' );
+
+		$debug_info .= "\r\n". esc_html( "Jetpack Sync Full Status: " . print_r( $sync_module->get_status(), 1 ) );
+
+		$next_schedules = wp_next_scheduled( 'jetpack_sync_full' );
+		if( $next_schedules ) {
+			$debug_info .= "\r\n". esc_html( "Next Jetpack Full Sync Schedule: " . date( 'r', $next_schedules ) );
```
Comment (jeherve) [inline class.jetpack-debugger.php]: Could you make all the strings translatable, so users can understand the debug results as well?

Thanks!

File: class.jetpack-debugger.php [hunk 2]
```diff
@@ -18,6 +18,27 @@ private static function is_jetpack_support_open() {
 		}
 	}
 
+	static function seconds_to_time( $seconds ) {
```
Comment (gravityrail) [inline class.jetpack-debugger.php]: I think it's simpler and less code just to use gmdate(), e.g.

```
gmdate("H:i:s", $seconds)
```

Comment (enejb) [inline class.jetpack-debugger.php]: true, but it doesn't give you info if the sync has been delayed for days or weeks even. Hopefully that shouldn't happen.

**Comment by gravityrail** [comment]
@enejb we should also include the lag and size of the full sync queue, since that's now separate from the incremental sync queue

**Comment by enejb** [comment]
Great point will add that!

File: class.jetpack-debugger.php [hunk 3]
```diff
@@ -60,6 +81,33 @@ public static function jetpack_debug_display_handler() {
 		$debug_info .= "\r\n" . esc_html( "SITE_URL: " . site_url() );
 		$debug_info .= "\r\n" . esc_html( "HOME_URL: " . home_url() );
 
+		$debug_info .= "\r\n";
+		require_once JETPACK__PLUGIN_DIR . 'sync/class.jetpack-sync-modules.php';
+		$sync_module = Jetpack_Sync_Modules::get_module( 'full-sync' );
+
+		$debug_info .= "\r\n". sprintf( esc_html__( 'Jetpack Sync Full Status: `%1$s`', 'jetpack' ), print_r( $sync_module->get_status(), 1 ) );
```
Comment (jeherve) [inline class.jetpack-debugger.php]: Is there any way we could get human readable times in here as well? This isn't very useful to most people:

```
[started] => 1469788563
[queue_finished] => 1469788563
[sent_started] => 1469788564
[finished] => 1469788564
```

Comment (enejb) [inline class.jetpack-debugger.php]: True. Good catch.

**Comment by gravityrail** [comment]
LGTM! 👍

**Comment by gravityrail** [comment]
Merged to 4.2

**Pull request merged**